.env

```diff
@@ -0,0 +1,5 @@
+# Docker environment configuration file
+# Do not commit/push changes to this file unless they
+# are changes/additions that everyone should recieve.
+Sa_Password=D0ckeIzKn!ght
```

Comment:
Is this common practice to commit this at all? Should this perhaps be a `.default` or an `.example` file?
Ideally we'd still need a default if not set, and I'm not sure how we'd do that consistently as it's used by both `docker-compose.yaml` and `entry.sh`.

I'm just concerned about people committing this file at all and leaking their password in doing so, even if we aren't merging it in.

Same goes for `kodb-util-config.yaml`, which I feel might be slightly easier since it's loaded from script, you can just copy & rename the default file if it doesn't already exist.